Source/WebCore/platform/graphics/gstreamer/GStreamerVideoFrameConverter.cpp

```diff
@@ -46,14 +46,14 @@ GStreamerVideoFrameConverter::GStreamerVideoFrameConverter()
     GST_DEBUG_CATEGORY_INIT(webkit_gst_video_frame_converter_debug, "webkitvideoframeconverter", 0, "WebKit GStreamer Video Frame Converter");
     m_pipeline = gst_element_factory_make("pipeline", "video-frame-converter");
     m_src = makeGStreamerElement("appsrc", nullptr);
+    auto glcolorconvert = makeGStreamerElement("glcolorconvert", nullptr);
     auto gldownload = makeGStreamerElement("gldownload", nullptr);
-    auto videoconvert = makeGStreamerElement("videoconvert", nullptr);
     auto videoscale = makeGStreamerElement("videoscale", nullptr);
     m_sink = makeGStreamerElement("appsink", nullptr);
     g_object_set(m_sink.get(), "enable-last-sample", FALSE, "max-buffers", 1, nullptr);
 
-    gst_bin_add_many(GST_BIN_CAST(m_pipeline.get()), m_src.get(), gldownload, videoconvert, videoscale, m_sink.get(), nullptr);
-    gst_element_link_many(m_src.get(), gldownload, videoconvert, videoscale, m_sink.get(), nullptr);
+    gst_bin_add_many(GST_BIN_CAST(m_pipeline.get()), m_src.get(), glcolorconvert, gldownload, videoscale, m_sink.get(), nullptr);
+    gst_element_link_many(m_src.get(), glcolorconvert, gldownload, videoscale, m_sink.get(), nullptr);
```

Comment:
This breaks webcodec tests, with glcolorconvert we no longer accept video/x-raw (gldownload does).

ERROR   webkitvideoframeconverter GStreamerVideoFrameConverter.cpp:97:convert:<video-frame-converter> Unable to convert video frame. Error: error message: 0x7fed34004b90, time 99:99:99.999999999, seq-num 227, element 'appsrc0', GstMessageError, gerror=(GError)NULL, debug=(string)"../libs/gst/base/gstbasesrc.c\(3177\):\ gst_base_src_loop\ \(\):\ /GstPipeline:video-frame-converter/GstAppSrc:appsrc0:\012streaming\ stopped\,\ reason\ not-negotiated\ \(-4\)", details=(structure)"details\,\ flow-return\=\(int\)-4\;";

Reply:
That's triggered from `GStreamerInternalVideoEncoder::encode()`... I don't remember in which situation we have GL there (hence the downloadSample call)...